src/02-crm/contacts.apib

```diff
@@ -73,6 +73,7 @@ Get a list of contacts.
                 + added_at: `2016-02-04T16:44:33+00:00` (string)
                 + updated_at: `2016-02-05T16:44:33+00:00` (string)
                 + `web_url`: `https://app.teamleader.eu/contact_detail.php?id=2a39e420-3ba3-4384-8024-fa702ef99c9f` (string)
+                + `national_identification_number`: BE 4283 (string)
```

Comment:
PP: it would be better to move this property somewhere after IBAN and BIC, before the more "preferency" properties. `web_url` is a metadata type of property so I wouldn't return more user provided entity properties after that one.

Reply:
Q: Would this also mean changing the order its returned in the actual response?

Reply:
Yes preferably.

Reply:
Added default value for `national_identification_number` under the field `bic`